editor/editor_node.cpp

```diff
@@ -103,6 +102,8 @@
 #include "editor/import/resource_importer_texture.h"
 #include "editor/import/resource_importer_texture_atlas.h"
 #include "editor/import/resource_importer_wav.h"
+#include "editor/import/scene_importer_mesh_3d.h"
```

Comment:
meshes can be 2D or 3D and both should theoretically be supported, so I suggest removing the 3D from ImporterMesh (but leave it for MeshNode3D)

Reply:
Done.